test/players.js

```diff
@@ -90,6 +90,45 @@ describe('GET /players', () => {
             })
             .expect(httpStatus.BAD_REQUEST);
     });
+
+    it('should return 5 players on the 2nd page', async () => {
+        const res = await supertest(app)
+            .get('/players')
+            .query({
+                pageNumber: 2,
```

Comment:
Let's call this param just page and limit, I think I've already written this comment